uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_hw/launch/drivetrain.launch.py

```diff
@@ -0,0 +1,54 @@
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.conditions import IfCondition
+from launch.substitutions import Command, FindExecutable, LaunchConfiguration, PathJoinSubstitution
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+
+from pathlib import Path
+
+
+def generate_launch_description():
+
+    robo = (
+        Path(
+            r"/home/niiquaye/robotics/install/uwrt_mars_rover_drivetrain_description/share/uwrt_mars_rover_drivetrain_description/urdf/robot.urdf"
```

Comment:
This is not portable. See launch files in #171

Reply:
oh ya I gotta remove this .... idk why I even have this in this commit